homeassistant/components/air_pollutants/opensensemap.py

```diff
@@ -0,0 +1,107 @@
+"""
+Support for openSenseMap Air Pollutants data.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/air_pollutants_opensnesemap/
```

Comment:
```suggestion
https://home-assistant.io/components/air_pollutants_opensensemap/
```

This is not a SNES console 😜

Reply:
Thanks